deploy/charts/harvester/dependency_charts/csi-snapshotter/values.yaml

```diff
@@ -3,5 +3,5 @@
 # Declare variables to be passed into your templates.
 image:
   repository: k8s.gcr.io/sig-storage/snapshot-controller
-  tag: v4.0.0
+  tag: v6.1.0
```

Comment:
u need to run `helm package update` to update the dependency chart and add the generated tgz file u have modified only.